### ISSUE TITLE
New version broke webpack build (interval-tree-1d)

### ISSUE THREAD

**Issue by Vanuan** [bvaughn/react-virtualized#646]
It looks like 9.6.0 published 2 hours ago broke things when we use webpack 
```
Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
    at Object.<anonymous> (intervalTree.js:16)
    at Object.<anonymous> (intervalTree.js:386)
    at __webpack_require__ (bootstrap 265b87b…:52)
    at Object.<anonymous> (PositionCache.js:1)
    at __webpack_require__ (bootstrap 265b87b…:52)
    at Object.<anonymous> (Masonry.js:1)
    at __webpack_require__ (bootstrap 265b87b…:52)
    at Object.<anonymous> (index.js:1)
    at __webpack_require__ (bootstrap 265b87b…:52)
    at Object.<anonymous> (index.js:1)
```

The problem is in this line:

https://github.com/bvaughn/react-virtualized/blob/master/source/vendor/intervalTree.js#L16

`module` is invalid syntax inside ES6 module.

**Comment by bvaughn** [comment]
Yeah, I vendored that file but missed the export. Will do a point release momentarily.

**Comment by bvaughn** [comment]
Fixed with 84bac59, released as 9.6.1

Thanks for reporting this
